docker/conf/docker-compose.github-actions.review.simplified.yml.dist

```diff
@@ -43,6 +43,12 @@ services:
             PACKETERY_API_KEY: PACKETERY_API_KEY_VALUE
             PACKETERY_API_PASSWORD: PACKETERY_API_PASSWORD_VALUE
             MESSENGER_TRANSPORT_VHOST: BRANCH_NAME
+            FACEBOOK_CLIENT_ID: FACEBOOK_CLIENTID
```

Comment:
please, add the same variables also into `project-base/gitlab/docker-compose-ci-review.yml` (for gitlab CI) and also to `project-base/app/deploy/deploy-project.sh`

Reply:
added in https://github.com/shopsys/shopsys/pull/3154/commits/0a793647744e39092349b7b9eaa03c12e93aa4a4